src/DynamoCore/BuiltInAndOperators/Operators.xml

```diff
@@ -110,9 +110,10 @@
         </member>
 		<member name="%Not">
 		            <summary>
-                !x
+                Reverses the result, returns false if the result is true
```

Comment:
I would suggest `Negate` instead of `Reverse`:
```
Negates the input, e.g. returns false when the input is true.
```

Reply:
Addressed in 209bb6c